protos/feast/core/FeatureSet.proto

```diff
@@ -41,9 +42,9 @@ message FeatureSetSpec {
     // List of features contained within this featureSet.
     repeated FeatureSpec features = 4;
 
-    // Features in this feature set will only be retrieved if they are found after [Time - MaxAge]
+    // Features in this feature set will only be retrieved if they are found after [retrievalTime - maxAge]
```

Comment:
This applies to historical data as well. So it's not just about the time of retrieval. We scan back from each event time that the user provides using maxAge.

Reply:
OK
I just feel `Time` here might be a bit ambiguous.

Reply:
It is a bit ambiguous, but not sure if we should be documenting the functionality of Feast within the protos. Is it better to remove it entirely?